tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,154 @@
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' truevalue='-bp' falsevalue=''/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' truevalue='-mb' falsevalue=''/>
+            <!--<param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots'
+                   argument='-sa' optional='true'/>/-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' truevalue='-hm' falsevalue=''/>
+            <param type='boolean' name='generate_clustermap' label='Flag to generate clustermap' truevalue='-cm' falsevalue=''/>
+        </section>
+        <section name='IP' title='Select interactive plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' truevalue='-IB' falsevalue=''/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' truevalue='-IM' falsevalue=''/>
+           <!-- <param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots' argument='-IS'
+                   optional='true'/>-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' truevalue='-HM' falsevalue=''/>
+        </section>
+        <param type='boolean' name='stack' label='Flag to unstack bars in barplots' truevalue='-s' falsevalue=''/>
+        <param type='boolean' name='verbosity' label='Flag to get debug information' truevalue='-v' falsevalue=''/>
+        <param type='boolean' name='annotation' label='Flag to add annotations on the maps' truevalue='-a' falsevalue=''/>
+    </inputs>
+    
+    <outputs>
+        <data name='data_output' format='csv' label='Formatted data used for plotting' />
+        <data name='plots_output' format='zip' label='Resulting plots' />
+        <data name='log_output' format='txt' label='Run Info' />
+    </outputs>
+    
+    <tests>
+        <test>
+    	  <param name='datafile' value='input_data.csv' />
+    	  <param name='template_file' value='input_template.xlsx'/>
+    	  <param name='format' value='pdf' />
+    	  <param name='value' value='isotopologue_fraction' />
+    	  <section name='SP' >
+    	   <param name='generate_barplot' value='-bp' />
+    	  </section>
+    	  <section name='IP' >
+    	   <param name='generate_barplot' value='-IB' />
+    	  </section>
+          <output name='data_output' file='data_out.csv'/>
+          <output name='plots_output'>
+            <assert_contents>
+                <has_size value='2852573' delta='1000' />
+            </assert_contents>
+            </output>
+    	  <output name='log_output'>
+            <assert_contents>
+                <has_text text='- isoplot_log.main.cli_process - INFO - Plots created. Run is terminated' />
+            </assert_contents>
+          </output>
+        </test>
+    </tests>  
+    
+    <help><![CDATA[

```

Comment:
Can you add a link to the isoplot documentation?
It could help to handle the input files.